include/uv-win.h

```diff
@@ -343,7 +343,9 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
   /* Threadpool */                                                            \
   void* wq[2];                                                                \
   uv_mutex_t wq_mutex;                                                        \
-  uv_async_t wq_async;
+  uv_async_t wq_async;                                                        \
+  /* Queue of all active uv_loops */                                          \
+  void* uv_loops[2];
```

Comment:
this would break the ABI. There should be 4 reserved void* fields already, let's use the first 2 for this.